linux/ibus-keyman/tests/meson.build

```
@@ -0,0 +1,158 @@
+test_files = [
+  'ibusimcontext.c',
+  'ibusimcontext.h',
+  'testfixture.cpp',
+  'testmodule.c',
+  'testmodule.h',
+]
+
+kmnkbp_tests_lib = cc.find_library(
+  'libkmnkbp-tests',
+  dirs: [
+    '@0@/../../../core/build/arch/debug/tests/unit/kmx'.format(meson.current_source_dir()),
+    '@0@/../../../core/build/arch/release/tests/unit/kmx'.format(meson.current_source_dir())
```

```suggestion
    core_dir / 'build/arch/debug/tests/unit/kmx',
    core_dir / 'build/arch/release/tests/unit/kmx'
```

Same, conditionalize on whether or not we are doing a debug build?